xapian-bindings/python3/Makefile.am

```diff
@@ -30,51 +30,72 @@ RST_DOCS = \
 # Therefore, we don't run it by default currently.
 # replicationtest.py
 
-BUILT_SOURCES = xapian_wrap.cc xapian_wrap.h xapian.py
+BUILT_SOURCES = xapian_wrap.cc xapian_wrap.h xapian.py \
+	xapianletor_wrap.cc \
+	xapianletor_wrap.h \
+	xapianletor.py
 
-EXTRA_DIST = python.i util.i extra.i extracomments.i except.i \
+EXTRA_DIST = python.i pythonletor.i util.i utilletor.i extra.i extracomments.i except.i \
+	exceptletor.i \
 	doxy2swig.py \
 	fixup-swig-py3-wrapper \
 	generate-python-exceptions \
+	generate-letor-python-exceptions \
 	run-python-test \
 	sphinx-build \
 	testsuite.py \
 	test_xapian_star.py \
 	$(TESTS) $(BUILT_SOURCES) $(RST_DOCS)
 
-pkgpylibdir = @PYTHON3_LIB@/xapian
+pkgpylibdir = @PYTHON3_LIB@/xapian \
+	@PYTHON3_LIB@/xapianletor
 
 # Install as _DATA rather than _SCRIPTS because we don't want to make these
 # executable (they don't have a #! line).
 pkgpylib_DATA = xapian/__init__.py \
 		xapian/__pycache__/__init__.@PYTHON3_CACHE_TAG@.pyc \
-		xapian/__pycache__/__init__.@PYTHON3_CACHE_OPT1_EXT@
+		xapian/__pycache__/__init__.@PYTHON3_CACHE_OPT1_EXT@ \
+		xapianletor/__init__.py \
+		xapianletor/__pycache__/__init__.@PYTHON3_CACHE_TAG@.pyc \
+		xapianletor/__pycache__/__init__.@PYTHON3_CACHE_OPT1_EXT@
 
-pkgpylib_LTLIBRARIES = _xapian.la
+pkgpylib_LTLIBRARIES = _xapian.la _xapianletor.la
 
 # Remove the .la file - _xapian.la is never linked against (it's a module)
 # and Python doesn't use libltdl.  Note that the library gets installed by
 # install-data, so that's where we need to hook.
 install-data-hook:
 	rm -f $(DESTDIR)$(pkgpylibdir)/_xapian.la
+	rm -f $(DESTDIR)$(pkgpylibdir)/_xapianletor.la
 
 # Because we don't install the .la file, "make uninstall" doesn't work and
 # we need to remove the file ourselves.
 uninstall-local:
 	rm -f $(DESTDIR)$(pkgpylibdir)/_xapian$(PYTHON3_SO)
+	rm -f $(DESTDIR)$(pkgpylibdir)/_xapianletor$(PYTHON3_SO)
```

Comment:
And these new lines need to use `letorpkgpylibdir` too.

Reply:
fixed in a fixup commit.